### PR TITLE
Post 2020-09-30 release tasks

### PR DESCRIPTION
**Pull request by hasufell**

### CODE REVIEW

File: nix/migration-tests.nix [hunk 1]
```diff
@@ -36,26 +36,12 @@ let
...
-    { rev = "v2020-06-05";
-      sha256 = "062l3i61074vlak44335mffr3xzpbs9nhsiis0rwlrn4w4wiwzx3"; }
-    { rev = "v2020-07-06";
-      sha256 = "1pl1vqmdyjx8ly3vy48j211hh59w7ikksmzv7r4y1cpjyi0ajjsa"; }
     { rev = "v2020-07-28";
       sha256 = "1mnnlg1x3y9cf3sqmxpqjdiwlay58pdci4cjxfvlwlyqqlsy5d1i"; }
     { rev = "v2020-08-03";
       sha256 = "1bh6mwlzc77x7ka2kihfbdgg1lwvrdb26280kvdznwxcf4nbjgmk"; }
+    { rev = "v2020-09-30";
+      sha256 = "????????????????????????????????????????????????????"; }
```
Comment (hasufell) [inline nix/migration-tests.nix]: I don't know

Comment (Anviking) [reply nix/migration-tests.nix]: _Perhaps_ you could find it in a similar way to what is described here: https://github.com/input-output-hk/cardano-wallet/blob/29a503815efe6b7cf9b2366faccb957169906737/nix/jormungandr.nix#L1-L30

Comment (Anviking) [reply nix/migration-tests.nix]: In which case ongoing CI build might tell you soon.

Comment (hasufell) [reply nix/migration-tests.nix]: > Perhaps you could find it in a similar way to what is described here: 

That's an excellent suggestion :smile:

Comment (KtorZ) [reply nix/migration-tests.nix]: Best way to figure this out is to put a random hash, build it with nix, and let nix tells you the hash it expected :upside_down_face:

Comment (hasufell) [reply nix/migration-tests.nix]: I can't:

```
$ nix-build nix/migration-tests.nix -o migration-tests                            32.1s  Thu 17:29
trace: Cleaning component source not supported for hpack package: cardano-addresses-2.0.0
trace: Cleaning component source not supported for hpack package: cardano-addresses-cli-2.0.0
trace: Cleaning component source not supported for hpack package: cardano-addresses-2.0.0
trace: Cleaning component source not supported for hpack package: cardano-addresses-cli-2.0.0
trace: Cleaning component source not supported for hpack package: cardano-addresses-2.0.0
trace: Cleaning component source not supported for hpack package: cardano-addresses-cli-2.0.0
trace: Cleaning component source not supported for hpack package: cardano-addresses-2.0.0
trace: Cleaning component source not supported for hpack package: cardano-addresses-cli-2.0.0
trace: Cleaning component source not supported for hpack package: cardano-addresses-2.0.0
trace: Cleaning component source not supported for hpack package: cardano-addresses-cli-2.0.0
trace: Cleaning component source not supported for hpack package: cardano-addresses-2.0.0
trace: Cleaning component source not supported for hpack package: cardano-addresses-cli-2.0.0
trace: Cleaning component source not supported for hpack package: cardano-addresses-2.0.0
trace: Cleaning component source not supported for hpack package: cardano-addresses-cli-2.0.0
error: unable to fork: Cannot allocate memory
(use '--show-trace' to show detailed location information)
```

It seems to be leaking memory.

File: nix/migration-tests.nix [hunk 2]
```diff
@@ -36,26 +36,12 @@ let
...
-    { rev = "v2020-07-28";
-      sha256 = "1mnnlg1x3y9cf3sqmxpqjdiwlay58pdci4cjxfvlwlyqqlsy5d1i"; }
-    { rev = "v2020-08-03";
-      sha256 = "1bh6mwlzc77x7ka2kihfbdgg1lwvrdb26280kvdznwxcf4nbjgmk"; }
+    { rev = "v2020-09-11";
+      sha256 = "0755ggsgnlg6p0pnnrcjnbwbfg61faiyjnzng3vsva2nvg3sxn20"; }
+    { rev = "v2020-09-22";
+      sha256 = "1nfzfdv03nz2bhin2jflxsmh597vkjifb76jsdg0384x89gi0a10"; }
+    { rev = "v2020-09-30";
+      sha256 = "19vmyq0m67ih295z1y8ng2rdn85mi0rvw26k711lbxsqri7fmyvz"; }
```
Comment (KtorZ) [inline nix/migration-tests.nix:58]: Wow. We certainly want to keep more history than the last 3 releases. Last 10 was what we agreed on in the past :)

File: README.md [hunk 3]
```diff
@@ -71,11 +71,10 @@ See **Installation Instructions** for each available [release](https://github.co
...
+> | [v2020-09-30](https://github.com/input-output-hk/cardano-wallet/releases/tag/v2020-09-30) | [v0.9.0](https://github.com/input-output-hk/jormungandr/releases/tag/v0.9.0) | [1.20.0](https://github.com/input-output-hk/cardano-node/releases/tag/1.20.0)
+> | [v2020-09-22](https://github.com/input-output-hk/cardano-wallet/releases/tag/v2020-09-22) | [v0.9.0](https://github.com/input-output-hk/jormungandr/releases/tag/v0.9.0) | [1.19.1](https://github.com/input-output-hk/cardano-node/releases/tag/1.19.1) | [v0.9.0](https://github.com/input-output-hk/jormungandr/releases/tag/v0.9.0) | [1.20.0](https://github.com/input-output-hk/cardano-node/releases/tag/1.20.0)
+> | [v2020-09-11](https://github.com/input-output-hk/cardano-wallet/releases/tag/v2020-09-11) | [v0.9.0](https://github.com/input-output-hk/jormungandr/releases/tag/v0.9.0) | [1.19.1](https://github.com/input-output-hk/cardano-node/releases/tag/1.19.1)
 > | [v2020-08-03](https://github.com/input-output-hk/cardano-wallet/releases/tag/v2020-08-03) | [v0.9.0](https://github.com/input-output-hk/jormungandr/releases/tag/v0.9.0) | [1.18.0](https://github.com/input-output-hk/cardano-node/releases/tag/1.18.0)
```
Comment (KtorZ) [inline README.md]: Here we only keep the last 3 releases in principle. So we can ditch 08-03

File: nix/migration-tests.nix (hunk 2 above)
Comment (hasufell) [reply nix/migration-tests.nix:58]: https://github.com/input-output-hk/cardano-wallet/wiki/Release-Checklist#publication says 3, I guess it needs updating

**Review by KtorZ — APPROVED** [review]
bors merge

**Pull request merged**